packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -29,20 +29,32 @@ export const pausingUploadReason = Symbol('pausing upload, not an actual error')
 class MultipartUploader {
   #abortController = new AbortController()
 
+  /** @type {import("../types/chunk").Chunk[]} */
   #chunks
 
+  /** @type {{ uploaded: number, etag?: string, done?: boolean }[]} */
   #chunkState
 
+  /**
+   * The (un-chunked) data to upload.
+   *
+   * @type {Blob}
+   */
   #data
 
+  /** @type {import("@uppy/core").UppyFile} */
   #file
 
-  #uploadPromise
+  /** @type {boolean} */
+  #uploadHasStarted = false
 
+  /** @type {(err?: Error | any) => any} */
```

Comment:
I guessing this is what we want?
```suggestion
  /** @type {(err?: Error | any) => void} */
```